compiler/x/codegen/X86BinaryEncoding.cpp

```diff
@@ -176,21 +176,36 @@ uint8_t getMemoryBarrierBinaryLengthLowerBound(int32_t barrier, TR::CodeGenerato
 
 // -----------------------------------------------------------------------------
 // OMR::X86::Instruction:: member functions
+bool OMR::X86::Instruction::needsLockPrefix()
```

Comment:
Why this helper for lockPrefix and not one for needsRepPrefix? Shouldn't we be consistent?
What is the value of this one-liner?